Gemfile

```diff
@@ -48,3 +48,9 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'devise'
```

Comment:
Move Devise and Bootstrap up with the other gems in the global space. It'll read better. :)